yang/ietf-l3vpn-ntw.yang

```diff
@@ -2345,6 +2353,38 @@ module ietf-l3vpn-ntw {
                     if-feature "vpn-common:bfd";
                     description
                       "Container for BFD.";
+                    leaf transmit_interval {
```

Comment:
Please change the names to point to the following defined in RFC5880

```
   |                    Desired Min TX Interval                    |
   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
   |                   Required Min RX Interval                    |
```
The description can be shortned to indicate that this corresponds to 
bfd.DesiredMinTxInterval & bfd.RequiredMinRxInterval defined in Section 6.8.7 of 5880.